frontend/src/metabase/query_builder/components/expressions/ExpressionWidgetInfo/ExpressionWidgetInfo.tsx

```diff
@@ -0,0 +1,37 @@
+import { t } from "ttag";
+import Tooltip from "metabase/core/components/Tooltip";
+import { useSelector } from "metabase/lib/redux";
+import { getShowMetabaseLinks } from "metabase/selectors/whitelabel";
+import { Box } from "metabase/ui";
+import { InfoLink, StyledFieldTitleIcon } from "../ExpressionWidget.styled";
+import { EXPRESSIONS_DOCUMENTATION_URL } from "../ExpressionWidget";
+
+export function ExpressionWidgetInfo() {
+  const showMetabaseLinks = useSelector(getShowMetabaseLinks);
+
+  return showMetabaseLinks ? (
```

Comment:
This is the exception where we don't completely hide the link but render it, but don't make it clickable and remove the copy `Click for documentation.` from the tooltip.